test/evergreen.yml

```diff
@@ -6106,10 +6096,7 @@ buildvariants:
     CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/TCMALLOC_LIB"
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
     additional_env_vars: |
-      export ASAN_OPTIONS="abort_on_error=1:detect_leaks=1:disable_coredump=0:unmap_shadow_on_exit=1"
-      export LSAN_OPTIONS="abort_on_error=1:print_suppressions=0:suppressions=$(git rev-parse --show-toplevel)/test/evergreen/asan_leaks.supp"
-      export ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v4/bin/llvm-symbolizer
-      export TESTUTIL_BYPASS_ASAN=1
+      export LSAN_OPTIONS="$COMMON_SAN_OPTIONS:print_suppressions=0:suppressions=$(git rev-parse --show-toplevel)/test/evergreen/asan_leaks.supp"
```

Comment:
Yeap! so this is okay because ${additional_env_vars} gets expanded at the last line of "setup environment", so at that point `COMMON_SAN_OPTIONS` is already defined.